aws-ecr-replicationconfiguration/.gitignore

```diff
@@ -0,0 +1,23 @@
+# macOS
```

Comment:
shouldn't this just be at the top level for all resources ?

Reply:
This is just the general practice for all CFN repos that I saw so I just kept it consistent.